viewer/src/components/SpecialAction.vue

```diff
@@ -52,19 +60,21 @@ export default class SpecialAction extends Vue {
       this.$emit("click");
       return;
     }
-    this.$store.dispatch("gaiaViewer/actionClick", this.action.join(","));
+    this.$store.dispatch("gaiaViewer/specialActionClick", this.action.join(","));
+  }
+
+  get tooltip() {
+    return this.board ? undefined : withShortcut(translateResources(this.rewards), this.shortcut);
   }
 
   get rewards() {
-    return new Event(this.action[0]).rewards;
+    return new Event('>' + this.action[0]).rewards;
```

Comment:
Maybe just `Rewad.parse(this.action[0])`

Reply:
I guess this is more something for the "events" pr